tagger/plot/basic.py

```diff
@@ -16,6 +23,12 @@
 pylab.rcParams.update(params)
 
 import os
+from .common import PT_BINS
+from scipy.stats import norm
+
+###### DEFINE ALL THE PLOPTTING FUNCTIONS HERE!!!! THEY WILL BE CALLED IN basic() function >>>>>>>
+def average(number1, number2):
+  return (number1 + number2) / 2.0
```

Comment:
Why does this need a separate function, is this significantly faster than np.mean or a built in function?